R/parameters.R

```diff
@@ -45,3 +66,21 @@ get_population <- function(iso3c) {
 get_country <- function(iso3c) {
   squire::population[squire::population$iso3c == iso3c, "country"][[1]]
 }
+
+#' Function to add asymptomatic information
+#'
+#' @return list of asymptomatic parameters
+get_asymptomatic <- function()
+{
+
+  prob_asymp <- c(0.3, 0.3, rep(0.2, 15))
```

Comment:
Can you add explanations or sources for these magic numbers so that if they need updating it's easier to spot them?